src/main/java/com/google/cloud/tools/gradle/appengine/AppEngineStandardPlugin.java

```diff
@@ -234,4 +238,23 @@ public void execute(Project project) {
       }
     });
   }
+
+  private void createShowConfigurationTask() {
```

Comment:
Looks like a significant amount of code duplication here. Can this method be factored out for reuse?